GeneralSQL/Error.py,sha256=kBrYZ17K0Jps9-DY3QFG_tLa9GOrLBJvoTLv8wwxfMk,2321
GeneralSQL/__init__.py,sha256=AviJrr-X4G1bh8717HsT4StS2eAI677ZgygSq8RsLLI,2155
GeneralSQL/Template/__init__.py,sha256=F80meUoOqQIOiYxE9ryCJzTMqFPBOwajkaf6Gk1e7Zk,1431
GeneralSQL/Template/base.py,sha256=oQTj7lN5JmlD3l3fG3-pm3dD8lPd0e6T1H5b0yOxToE,3024
GeneralSQL/db/Jmysql.py,sha256=scf96hhIKFhjRRpzjqO6Qckn4yenS1nGBY2baBQYDXA,15659
GeneralSQL/db/__init__.py,sha256=PgWm-6m5ZDVt2vjNOtrgO7sQYzXa6A0p8Y2RnoZNTl0,57
GeneralSQL-1.1.7.dist-info/METADATA,sha256=RdGlwh0XfmDmlNlmW8yp8hz7x_6h4_y-xKZ1a8MXtSk,42392
GeneralSQL-1.1.7.dist-info/WHEEL,sha256=oiQVh_5PnQM0E3gPdiz09WCNmwiHDMaGer_elqB3coM,92
GeneralSQL-1.1.7.dist-info/top_level.txt,sha256=N28y8Tp5KNt4vFIdlg6UXzrY9QGtXwgK-oqRvdRBuh8,11
GeneralSQL-1.1.7.dist-info/RECORD,,
